tests/timescaledb/copydb.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+set -x
+set -e
+
+# This script expects the following environment variables to be set:
+#
+#  - PGCOPYDB_SOURCE_PGURI
+#  - PGCOPYDB_TARGET_PGURI
+#  - PGCOPYDB_TABLE_JOBS
+#  - PGCOPYDB_INDEX_JOBS
+
+# make sure source and target databases are ready
+pgcopydb ping --source ${PGCOPYDB_SOURCE_PGURI} --target ${PGCOPYDB_TARGET_PGURI}
```

Comment:
I didn't understand why PGCOPYDB_SOURCE_PGURI and PGCOPYDB_TARGET_PGURI being used explicitly? Removing them should work fine.

Reply:
Yes, I have removed the URLs.